apis/r/NEWS.md

```diff
@@ -1,11 +1,22 @@
-# tiledbsoma (development version)
+# tiledbsoma 0.1.19
+
+## Changes
+
+- Updated `setup-r` GitHub Action to v2
+
+# tiledbsoma 0.1.18
 
 ## Features
 
 - The `AnnotationMatrix`'s `to_matrix()` method now supports batched reads via the `batch_mode` argument. This functionality can also be leveraged from `SOMA`'s  `get_seurat_dimreductions_list()` and `get_seurat_dimreduction()` methods.
 
+## Changes
+
+- Increased capacity of `AssayMatrix` arrays from 1,000 to 100,000 to improve remote-read performance (#543)
```

Comment:
This was a move from 100,000 down to 1,000 in order to increase read-locality for point queries

Reply:
Good catch!